### ISSUE TITLE
How to follow redirects?

### ISSUE THREAD

**Issue by 9mm** [petrohi/machine_gun#9]
Hey, just trying to figure out how I can follow 301's instead of returning that. Thanks

**Comment by petrohi** [comment]
Yes. Both MachineGun and Gun are low level HTTP libraries. Gun is connection and protocol level and MachineGun is pool level on top of Gun. Higher level implementation is needed to support "fetch" level including retries and redirects.
